packages/react/src/primitives/Icon/icons/IconAdd.tsx

```diff
@@ -1,13 +1,14 @@
-import classNames from 'classnames';
 import * as React from 'react';
+import classNames from 'classnames';
```

Comment:
We have a script to generate the icons. We need to update the script as well <- https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/scripts/generateIcons.js

Reply:
Removed the script :) https://github.com/aws-amplify/amplify-ui/pull/3209